builtin/logical/mysql/path_roles.go

```diff
@@ -34,6 +34,11 @@ func pathRoles(b *backend) *framework.Path {
 				Type:        framework.TypeString,
 				Description: "SQL string to create a user. See help for more info.",
 			},
+
+			"username_length": &framework.FieldSchema{
```

Comment:
@vishalnayak okay, just seem redundant to provide for each role, when the underlying connection (and desired length) probably are the same anyway ?